src/samplers/adapt_sampler.js

```diff
@@ -0,0 +1,116 @@
+// @flow
```

Comment:
May want to call it _v1_to_v2_adapter (private module)

Reply:
SGTM.

Reply:
Renamed to `_adapt_sampler.js` because the export is a factory function.